vista_2d/vista_2d_tutorial_monai.ipynb

```diff
@@ -0,0 +1,1405 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    !pip install git+https://github.com/facebookresearch/segment-anything.git
</pre><p>!python -c "import segment_anything " || pip install git+<a href="https://github.com/facebookresearch/segment-anything.git" target="_blank">https://github.com/facebookresearch/segment-anything.git</a></p><p> </p><p>Could you please modify the import area here to check whether the package can be imported first then to install if not?</p><p>Like:</p><p>!python -c "import segment_anything " || pip install git+<a href="https://github.com/facebookresearch/segment-anything.git" target="_blank">https://github.com/facebookresearch/segment-anything.git</a></p><p> </p><p>Refer:</p><p><a href="https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb" target="_blank">https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/spleen_segmentation_3d.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1788/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Sure thing, addressed.